### ISSUE TITLE
Markdown linting

### ISSUE THREAD

**Issue by mfulton26** [denoland/deno_lint#1292]
Is Markdown linting out of scope? I think it would be nice to have `deno lint` address all linting concerns for things that Deno supports/consumes/etc. `deno fmt` supports Markdown so I think it would be nice if `deno lint` were to support it too.

There might be some opportunity to reuse and/or learn from https://github.com/DavidAnson/markdownlint-cli2 and https://github.com/DavidAnson/vscode-markdownlint.

**Comment by bartlomieju** [comment]
I wouldn't say it's out of scope, but the Deno team doesn't have bandwidth to work on this currently. I'll be more than happy to accept contributions though.

**Comment by mfulton26** [comment]
> I wouldn't say it's out of scope, but the Deno team doesn't have bandwidth to work on this currently. I'll be more than happy to accept contributions though.

I haven't looked yet at any implementation details of `deno lint` yet. Based on the language usage in the GitHub repository I'm guessing most if not all linting is done in Rust code. Is there a strong preference to keep everything in Rust and/or not depend on 3rd part code from existing Markdown linters? My guess is yes but I don't want to assume. Thank you.

**Comment by bartlomieju** [comment]
Yes, the current preference is that these rules should be implemented in Rust. You might want to wait until plugin support is added (eye-balling to do it after Deno 2) which should allow to integrate JS based rules.
